spec/order_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'order'
+
+describe Order do 
+
+  test_menu = Takeaway.new.menu
```

Comment:
I believe you're supposed to use doubles here to create a fake Takeaway.new but seems like it would be much more code than this so not sure..?